### ISSUE TITLE
[BUG] Unable to Deploy NeuVector on RKE2 v1.25+ via the App Catalog

### ISSUE THREAD

**Issue by zackbradys** [rancher/charts#2917]
### Rancher Server Setup
- Rancher version: **v2.7.6**
- Installation option (Docker install/Helm Chart): **Helm via Rancher App Catalog**
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): **RKE2**
- Proxy/Cert Details: **Let's Encrypt**

### Information about the Cluster
- Kubernetes version: **v1.25.12+rke2r1**
- Cluster Type (Local/Downstream): **Local or Downstream**

### User Information
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions: **Admin (full permissions)**

### Provider Information
- What is the version of the Rancher v2 Terraform Provider in use? **No**
- What is the version of Terraform in use? **No**

### Describe the bug
Unable to deploy NeuVector 5.2.0 (102.0.3+up2.6.0) with the Rancher App Catalog on RKE2 v1.25 Cluster(s). A few work arounds include manually edit the yaml to include `global.cattle.psp.enabled: false` before installing and deploying or to use the older chart for NeuVector 5.1.3 (102.0.2+up2.4.5).

### To Reproduce / Actual Result
Use the Rancher App Catalog to try and deploy the latest NeuVector Chart 102.0.3+up2.6.0 on an RKE2 v1.25 Cluster. It will fail with the following error message:

```bash
2023-08-31T02:04:09.934128671Z helm upgrade --install=true --namespace=cattle-neuvector-system --timeout=10m0s --values=/home/shell/helm/values-neuvector-102.0.3-up2.6.0.yaml --version=102.0.3+up2.6.0 --wait=true neuvector /home/shell/helm/neuvector-102.0.3-up2.6.0.tgz
2023-08-31T02:04:09.971970604Z Release "neuvector" does not exist. Installing it now.
2023-08-31T02:04:10.046164921Z Error: template: neuvector/templates/validate-psp-install.yaml:2:14: executing "neuvector/templates/validate-psp-install.yaml" at <.Values.global.cattle.psp.enabled>: nil pointer evaluating interface {}.enabled
```

### Expected Result
Successful installation and deployment of NeuVector on the cluster.

### Additional context
Customers should be able to use the Rancher App Catalog to successfully deploy and install the latest and greatest versions of our products, especially customers utilizing the bundled charts inside of airgaps. Additionally, the ease of the Rancher App Catalog helps drive more customers to Rancher. Please let me know if you have any additional questions.

**Comment by zackbradys** [comment]
FYI @dweomer @clemenko @brooksphilip

**Comment by zackbradys** [comment]
Additionally, manually adding the `rancher-chart` repo at the `dev-2.7` branch and trying to deploy and install `102.0.4-up2.6.2` produces the same error:

```bash
helm upgrade --install=true --namespace=cattle-neuvector-system --timeout=10m0s --values=/home/shell/helm/values-neuvector-102.0.4-up2.6.2.yaml --version=102.0.4+up2.6.2 --wait=true neuvector /home/shell/helm/neuvector-102.0.4-up2.6.2.tgz
Thu, Aug 31 2023 11:56:26 am
Release "neuvector" does not exist. Installing it now.
Thu, Aug 31 2023 11:56:26 am
Error: template: neuvector/templates/validate-psp-install.yaml:2:14: executing "neuvector/templates/validate-psp-install.yaml" at <.Values.global.cattle.psp.enabled>: nil pointer evaluating interface {}.enabled
```

**Comment by zackbradys** [comment]
Updated and fixed with NeuVector 5.2.1 (102.0.4-up2.6.2) and related merge https://github.com/rancher/charts/pull/2940!
